plans/rpminspect/main.fmf

```diff
@@ -0,0 +1,28 @@
+summary: Run rpminspect on all srpm and binary rpms
```

Comment:
Maybe, we can have subdirectories for "categories":

* rpm / packaging (rpminspect, rpmlint, installability ...)
* security (e.g. csmock)

Or, maybe group `rpminspect` and `rpmlint` as linters?

Reply:
Yep I was considering a similar structure. I was also thinking about having a common group that calls both `rpminspect` and `rpmlint`, but not sure how to place those to distinguish them from the "component" plans. Maybe bundles will only be in top-level plans?